ld-pruning-wf.wdl

```diff
@@ -0,0 +1,440 @@
+version 1.0
+
+# General notes:
+#
+# variables and functions exclusive to the WDL version tend to have "py_" in front
+# and use camelCase between words - this is to make it clear they don't quite 
+# correlate to the CWL
+
+# [1] ld_pruning -- calculates linkage diseq on a GDS file
+task ld_pruning {
+	input {
+		File gds
+		File? sample_include_file
+		File? variant_include_file
+		String genome_build = "hg38"  # can also be hg18 or hg19
+		Float ld_r_threshold = 0.32
+		Float ld_win_size = 10  # yes, a float, not an int
+		Float maf_threshold = 0.01
+		Float missing_threshold = 0.01
+		#Boolean autosome_only = true  # buggy and not in CWL
+		Boolean exclude_pca_corr = true  # will act as String in Python
+		String? out_prefix
+		
+		# runtime attributes
+		Int addldisk = 1
+		Int cpu = 2
+		Int memory = 4
+		Int preempt = 3
+	}
+	command {
+		set -eux -o pipefail
+
+		echo "Generating config file"
+		# noquotes = argument has no quotes in the config file to match CWL config EXACTLY
+		python << CODE
+		import os
+		f = open("ld_pruning.config", "a")
+		f.write('gds_file "~{gds}"\n')
+		if "~{exclude_pca_corr}" != "true":
```

Comment:
We discussed this on Zoom but it's such a good point I'm going to essentially xpost that discussion on Github and link this to #14.

The CWL's logic seems to the reason behind its apparent bug. Most of the time `if(inputs.parameter)` is essentially the same as checking if something is defined. But because this is a boolean, I think the value of that boolean is influencing it.

In the CWL...
* If the user does not set `exclude_pca_corr` then the expression evaluates to `false` and the input not written to the config.
* If the user sets `exclude_pca_corr` to false then the expression evaluates to `false` and the input is not written to the config. This would result in the R script treating `exclude_pca_corr` as true even though it has been set as false, hence why I'm thinking this is a bug.
* The user is unlikely to set `exclude_pca_corr` to true as that is considered the default value.